jujugui/static/gui/src/app/app.js

```diff
@@ -452,6 +452,10 @@ YUI.add('juju-gui', function(Y) {
       @param {Object} controllerAPI The controller api instance.
     */
     _init: function(cfg, modelAPI, controllerAPI) {
+      // Store the initial model UUID.
+      const modelUUID = this.get('modelUUID') ||
+          (window.juju_config && window.juju_config.jujuEnvUUID);
```

Comment:
Do we have time to change this in core so that we use the modelUUID key instead?